package.json

```diff
@@ -49,128 +49,83 @@
+    "@storybook/addon-interactions": "^7.1.1",
+    "@storybook/addon-links": "^7.1.1",
     "@storybook/addon-styling": "^1.3.4",
-    "@storybook/addons": "^7.0.26",
-    "@storybook/blocks": "^7.0.26",
-    "@storybook/manager-api": "^7.1.0",
-    "@storybook/node-logger": "^7.0.26",
-    "@storybook/react": "^7.0.26",
-    "@storybook/react-webpack5": "^7.0.26",
+    "@storybook/addons": "^7.1.1",
+    "@storybook/blocks": "^7.1.1",
+    "@storybook/manager-api": "^7.1.1",
+    "@storybook/react": "^7.1.1",
+    "@storybook/react-webpack5": "^7.1.1",
     "@storybook/testing-library": "^0.2.0",
-    "@storybook/theming": "^7.1.0",
-    "@svgr/rollup": "^8.0.1",
-    "@svgr/webpack": "^8.0.1",
+    "@storybook/theming": "^7.1.1",
     "@testing-library/dom": "^9.3.1",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^14.0.0",
-    "@testing-library/react-hooks": "^8.0.1",
     "@testing-library/user-event": "^14.4.3",
-    "@types/bluebird": "^3.5.38",
-    "@types/jest": "^27.4.1",
+    "@types/jest": "^29.5.3",
     "@types/node": "^20.4.1",
     "@types/react": "^18.2.14",
     "@types/react-dom": "^18.2.6",
-    "@typescript-eslint/eslint-plugin": "^5.61.0",
-    "@typescript-eslint/parser": "^5.61.0",
-    "babel-core": "^6.26.3",
+    "@typescript-eslint/eslint-plugin": "^6.2.0",
+    "@typescript-eslint/parser": "^6.2.0",
     "babel-jest": "^29.3.1",
-    "babel-loader": "^8.2.5",
-    "babel-plugin-react-docgen": "^4.2.0",
-    "babel-plugin-react-require": "^3.1.1",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.24.1",
+    "babel-plugin-react-require": "^4.0.1",
     "boxt": "^1.1.1",
     "camelcase": "^7.0.1",
-    "chalk": "^4.1.2",
+    "chalk": "^5.3.0",
     "css-loader": "^6.8.1",
     "eslint": "^8.44.0",
     "eslint-config-prettier": "^8.8.0",
     "eslint-config-react-app": "^7.0.1",
-    "eslint-import-resolver-webpack": "^0.13.2",
-    "eslint-plugin-flowtype": "^8.0.3",
-    "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-jest": "^27.2.2",
-    "eslint-plugin-jsx-a11y": "^6.7.1",
-    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-react": "^7.32.2",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-storybook": "^0.6.12",
-    "execa": "^5.1.1",
-    "file-loader": "^6.2.0",
+    "eslint-plugin-storybook": "^0.6.13",
+    "execa": "^7.2.0",
     "gh-pages": "^5.0.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^27.5.1",
-    "jest-environment-jsdom": "^27.5.1",
-    "jest-pnp-resolver": "^1.2.3",
-    "jest-watch-typeahead": "^1.1.0",
+    "jest": "^29.6.1",
+    "jest-environment-jsdom": "^29.6.1",
     "lerna-changelog": "^2.2.0",
     "monday-ui-style": "^0.1.172",
     "postcss": "^8.4.25",
     "postcss-scss": "^4.0.6",
-    "prettier": "^2.8.8",
+    "prettier": "^3.0.0",
     "process": "^0.11.10",
     "prop-types": "^15.8.1",
```

Comment:
Are we using this?

Reply:
Not yet, but maybe in the future... as we probably don't want to convert this components to TS, but do want to export props types outside